backend/dal.keycloak/Pims.Dal.Keycloak.csproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <OutputType>Library</OutputType>
-    <Version>0.1.0.0-alpha</Version>
-    <AssemblyVersion>0.1.0.0</AssemblyVersion>
+    <Version>1.0.0.0</Version>
```

Comment:
everything else is 1.10.0.0 but this is 1.0.0.0?

Reply:
It would be nice to automate this in a build some how, but I doubt we'll have time to dedicate to it.